ui/components/config/core-settings-list.tsx

```
@@ -104,6 +122,26 @@ export default function CoreSettingsList() {
+	const handleLogQueueSizeChange = useCallback(
+		(value: string) => {
+			setLocalValues((prev) => ({ ...prev, log_queue_size: value }));
+
+			// Clear existing timeout
+			if (logQueueSizeTimeoutRef.current) {
+				clearTimeout(logQueueSizeTimeoutRef.current);
+			}
+
+			// Set new timeout
+			logQueueSizeTimeoutRef.current = setTimeout(() => {
+				const numValue = parseInt(value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)